package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node .",
-    "test": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test mocha --recursive",
```

Comment:
Do you have mocha installed globally? I had to add it to the package.json - `npm install -S -D mocha` to be able to run `npm test` on my local machine